dask/array/tests/test_array_core.py

```diff
@@ -414,6 +431,7 @@ def test_norm():
     assert b.vnorm(ord=4, axis=0, keepdims=True).ndim == b.ndim
```

Comment:
That would work.  If we wanted to be even lazier is it possible that we don't actually need to make dask.array pass 1.6 tests but rather than we only need dask.array to be importable?  Importable is a much weaker condition.